### ISSUE TITLE
distrobox should be installed by default along with toolbox

### ISSUE THREAD

**Issue by sudopluto** [sudopluto/unofficial-opensuse-microos-bug-tracker-wiki#8]
fix with t-u pkg install distrobox

**Comment by sysrich** [comment]
no, distrobox should be installed instead of toolbox on the MicroOS Desktop
and toolbox should remain the default on regular MicroOS

fix with actually fixing this in the patterns.
